web/public/locales/en.json

```diff
@@ -63,8 +63,8 @@
     "electricityproduction": "Electricity production",
     "electricityconsumption": "Electricity consumption",
     "emissions": "Carbon emissions",
-    "addeditsource": "<a href=\"{{link}}\" target=\"_blank\">add or edit source</a>",
```

Comment:
I would like to see this removed from all languages, but it doesn't need to be done in this PR if you wish to do it separately.

Reply:
I'd say it's not that important for things like this. It is more important to remove the unused things because that we will forget for sure. These will be translated again eventually.

We used to have a script for translating stuff but it's not really working and was very hacky to begin with. I have been planning to look into better options (there are some pre made solutions) but it's low on the priority list.